packages/sui-lint/eslintrc.js

```diff
@@ -83,8 +81,11 @@ module.exports = {
   rules: {
     ...REACT_RULES,
     ...TESTING_RULES,
-    'accessor-pairs': RULES.OFF,
     '@babel/no-unused-expressions': RULES.OFF,
+    'accessor-pairs': RULES.OFF,
+    'array-callback-return': RULES.WARNING,
```

Comment:
Moving to warning some new errors to avoid noise when upgrading to the new linter.